src/vs/vscode.proposed.d.ts

```diff
@@ -1,3 +1,5 @@
+import { ICommandEvent } from 'vs/platform/commands/common/commands'
```

Comment:
No dependencies allowed here, the Event type must be defined again, e.g. `interface ExecuteCommandEvent { command:string, args?:any[] }`